tests/unit/assessment/test_clusters.py

```diff
@@ -156,3 +162,40 @@ def test_unsupported_clusters():
     result_set = list(crawler.snapshot())
     assert len(result_set) == 1
     assert result_set[0].failures == '["cluster type not supported : LEGACY_PASSTHROUGH"]'
+
+
+def test_policy_crawler():
+    ws = workspace_client_mock(
+        policy_ids=['single-user-with-spn', 'single-user-with-spn-policyid', 'single-user-with-spn-no-sparkversion'],
+    )
+
+    crawler = PoliciesCrawler(ws, MockBackend(), "ucx")
+    result_set = list(crawler.snapshot())
+    assert len(result_set) == 2
+
+
+def test_policy_try_fetch():
+    ws = workspace_client_mock(policy_ids=['single-user-with-spn-policyid'])
+    mock_backend = MagicMock()
+    mock_backend.fetch.return_value = [("000", "test_policy", 1, "[]", "13.3x", "test", "abc")]
+    crawler = PoliciesCrawler(ws, mock_backend, "ucx")
+    result_set = list(crawler.snapshot())
+
+    assert len(result_set) == 1
+    assert result_set[0].policy_id == "000"
+    assert result_set[0].policy_name == "test_policy"
+    assert result_set[0].spark_version == "13.3x"
+    assert result_set[0].policy_description == "test"
+    assert result_set[0].creator == "abc"
+
+
+def test_policy_failure():
+    ws = workspace_client_mock(
+        policy_ids=['single-user-with-spn-policyid'],
+    )
+
+    with patch("databricks.labs.ucx.assessment.clusters.azure_sp_conf_present_check", retrun_value=True):
```

Comment:
this is prohibited. see https://github.com/databrickslabs/ucx/pull/1037

Reply:
Removed patching method